Makefile.in

```diff
@@ -63,8 +63,9 @@ create-dirs:
 
 install: all create-dirs
 	cd dmtcp && make DESTDIR=$(DESTDIR) $(INSTALL_FLAGS)
-	cd mpi-proxy-split && make DESTDIR=$(DESTDIR) $(INSTALL_FLAGS)
-	cd manpages && make DESTDIR=$(DESTDIR) $(INSTALL_FLAGS)
+	cd mpi-proxy-split && make DESTDIR=$(prefix) $(INSTALL_FLAGS)
```

Comment:
We shouldn't set DESTDIR to `prefix`. These are two separate variables and typically the final location is `$(DESTDIR)$prefix`